src/components/UI/theme.js

```diff
@@ -18,6 +18,9 @@ const theme = {
     med: '825px',
     large: '1200px',
   },
+  fontFamilies: {
+    console: 'Inconsolata, monospace',
+  },
```

Comment:
As I understand it, the naming here is up to us, right?

Reply:
Yes, we can use the naming we want